sqrl-engines/sqrl-engine-core/src/main/java/com/datasqrl/engine/EngineCapability.java

```diff
@@ -34,7 +34,9 @@ public enum EngineCapability {
   //Writing/upserting data into engine by primary key will deduplicate data
   MATERIALIZE_ON_KEY,
   //Engine supports data monitoring
-  DATA_MONITORING;
+  DATA_MONITORING,
+  //No capabilities
+  NONE;
```

Comment:
Why do you need that? I am not a big fan of having a capbility NONE - that's confusing.

Reply:
```
@NonNull EnumSet<EngineCapability> capabilities
```
It's not null and an empty EnumSet is not allowed.

Reply:
Ah, maybe there is one, let me check.